docs/tutorials/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>This section reviews the main charts and statistics that can be found on the platform dashboard. See  <a href="../monitoring/model-monitoring-deployment.html" target="_blank">Model monitoring overview</a> if you want to learn more about the available model monitoring features and how to use them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>